### PR TITLE
Mock

### PR DESCRIPTION
**Pull request by fepitre**

### CODE REVIEW

File: blivet/python-blivet.spec [hunk 1]
```diff
@@ -15,7 +11,7 @@ License: LGPLv2+
 Group: System Environment/Libraries
 %global realname blivet
 %global realversion %{version}%{?prerelease}
-Source0: http://github.com/rhinstaller/blivet/archive/%{realname}-%{realversion}.tar.gz
+Source0: %{realname}-%{realversion}.tar.gz
```
Comment (marmarek) [inline blivet/python-blivet.spec]: I'd leave URL here. If nothing more, as an info how to get new upstream version (`spectool --get-files` do that). You already have `(notdir ...)` in `SOURCE_ARCHIVE_NAME`, so it should just work, no?

**Comment by fepitre** [comment]
Following your comment, I kept the URL. I did it in two times initially, used notdir and then removed the URL but of course, it works well with the URL.

File: pungi/pungi.spec [hunk 2]
```diff
@@ -11,7 +7,7 @@ Summary:        Distribution compose tool
 Group:          Development/Tools
 License:        GPLv2
 URL:            https://pagure.io/pungi
-Source0:        https://pagure.io/releases/%{name}/%{name}-%{version}.tar.bz2
+Source0:        %{name}-%{version}.tar.bz2
```
Comment (marmarek) [inline pungi/pungi.spec]: URL here too.

**Review by marmarek — CHANGES_REQUESTED** [review]

File: pungi/pungi.spec (hunk 2 above)
Comment (fepitre) [reply pungi/pungi.spec]: Done.

**Comment by marmarek** [comment]
It can't find pykickstart-2.32.tar.gz while building src.rpm - it's looking for it in the wrong directory. Build log: https://gist.github.com/marmarek/a8c0b08e76f23bcefc9513f9aaf2178a

**Comment by marmarek** [comment]
I had outdated builder and builder-rpm, with new one it works.

**Review by marmarek — APPROVED** [review]

**Pull request merged**